src/main/java/net/neoforged/neoforge/common/ItemAbilities.java

```diff
@@ -120,17 +125,23 @@ public class ItemAbilities {
      */
     public static final ItemAbility BRUSH_BRUSH = ItemAbility.get("brush_brush");
 
+    /**
+     * Passed onto {@link IBlockExtension#getToolModifiedState} when a Flint and Steel wants to light a campfire/candle/cake
+     */
+    public static final ItemAbility FLINT_LIGHT = ItemAbility.get("flint_light");
```

Comment:
`FireChargeItem` is also capable of lighting, so it should also have this item ability, and thus instead of `FLINT_LIGHT`, we should probably have a more generic name here.

Reply:
Good point, how did I miss that? In addition, dispensers using Flint&Steel also properly light candles etc. So they should use the same mechanism, I guess.

About the name, as far as I see, all abilities are specific to one type of vanilla item at the moment. For both; which item has the ability as well as the name (the name has one exception: "till"). I wonder if any modded block/item may want to differentiate between "lighting like flint&steel" and "lighting like fire charge (as item)"? (Note: "lighting like fire charge (as entity)" already is something completely different and is handled by the same blocks that can be lit with the item.)

I tend to not splitting it, but what about the name? "light" would be useless without context. What about "kindle", "alight", "aflame" or "ignite"?

Reply:
"Aflame" is not a verb, "alight" is a verb but when used as a verb means something completely different. "Kindle" and "ignite" aren't bad in that sense, but saying you "kindle a cangle" or "ignite a candle" feels off even if it's technically correct. That said, I think "light" is also fine -- the immediate connotation here is "fire" and the javadoc will clarify any confusion.